components/suggestion/at_mention_provider/at_mention_suggestion.jsx

```diff
@@ -115,10 +115,12 @@ export default class AtMentionSuggestion extends Suggestion {
                 <span className='mention--align'>
                     {'@' + username}
                 </span>
-                <BotBadge
-                    show={Boolean(user.is_bot)}
-                    className='badge-autocomplete'
-                />
+                <div>
```

Comment:
What is this extra div needed for? You removed one from the ChannelHeader and added one here, so I'm unsure what it's needed for